### ISSUE TITLE
Why does VkPhysicalDeviceRayTracingMotionBlurFeaturesNV extend VkPhysicalDeviceProperties2 and not VkPhysicalDeviceFeatures2

### ISSUE THREAD

**Issue by expipiplus1** [KhronosGroup/Vulkan-Docs#1565]
Why does VkPhysicalDeviceRayTracingMotionBlurFeaturesNV extend VkPhysicalDeviceProperties2 and not VkPhysicalDeviceFeatures2?

```xml
        <type category="struct" name="VkPhysicalDeviceRayTracingMotionBlurFeaturesNV" structextends="VkPhysicalDeviceProperties2,VkDeviceCreateInfo">
            <member values="VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_RAY_TRACING_MOTION_BLUR_FEATURES_NV"><type>VkStructureType</type> <name>sType</name></member>
            <member optional="true">const <type>void</type>*        <name>pNext</name></member>
            <member><type>VkBool32</type>                           <name>rayTracingMotionBlur</name></member>
            <member><type>VkBool32</type>                           <name>rayTracingMotionBlurPipelineTraceRaysIndirect</name></member>
        </type>
```

All other `Vk*Features` structs (products of `VkBool32`) extend `VkPhysicalDeviceFeatures2`. Additionally in the SPIR-V capabilities it references the "`feature`" `rayTracingMotionBlur`.

```xml
        <spirvcapability name="RayTracingMotionBlurNV">
            <enable struct="VkPhysicalDeviceRayTracingMotionBlurFeaturesNV" feature="rayTracingMotionBlur" requires="VK_NV_ray_tracing_motion_blur"/>
        </spirvcapability>
```

**Comment by dgkoch** [comment]
Seems like a bug. Will fix.

**Comment by expipiplus1** [comment]
thanks!

**Comment by oddhack** [comment]
This should be fixed in the 1.2.183 spec update.
